ansible_collections/arista/avd/docs/porting-guides/5.x.x.md

```diff
@@ -204,6 +204,11 @@ It can be added if required by using `config_comment`:
 +config_comment: "RANCID-CONTENT-TYPE: arista"
 ```
 
+### Duplicate value of `address` is allowed in `router_pim_sparse_mode.ipv4.rp_addresses`
+
+In AVD 4.0.0, the value of `address` key was unique in `router_pim_sparse_mode.ipv4.rp_addresses`.
+In AVD 5.0.0, duplicate value of `address` is allowed in `router_pim_sparse_mode.ipv4.rp_addresses`.
+
```

Comment:
Thank you for being thorough, but for this change there is no changes required from the user, so we don't need an entry in the porting-guide.

```suggestion
```